neuror/cut_plane/detection.py

```diff
@@ -132,6 +133,9 @@ def find(cls, neuron, bin_width=3,
         Returns:
             A cut plane object
         """
+        warnings.warn("This method will be deprecated in favor or cut_leaves.find_cut_leaves"
```

Comment:
I think you can simply drop all my shit

Reply:
morphology-repair-workflow depends on it, no? Maybe better to test this new version a bit more, and make another PR to remove that + update morphokogy-repairr-workflow?

Reply:
Ok, ok, yes I'm fine with this.